### DISCUSSION TITLE
Festa a Fantasia

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#376] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/festa-a-fantasia/

**Comment by angellisy** [comment]
Tadinho do menino

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/festa-a-fantasia/#comment_1087464866>):

> Tadinho do menino

É mas... a ideia do paizão aí foi genial
